.github/workflows/build-beta-packtracker.yml

```diff
@@ -0,0 +1,59 @@
+name: Build Beta + Packtracker
+
+# Packtracker needs to be run on push events so we can't add it to the PR Build.
```

Comment:
I am still 100% convinced that we can do this correctly using a single pull_request workflow.

Reply:
I had another PR yesterday where I had it in a pull_request event. Pack tracker doesn't report properly in that event and I think it is because git commands don't work properly and I am not entirely sure why. Reason I think this is pack tracker can't just pull out the branch name and commit sha via git commands. This probably means it cannot get the prior commit either.